email-client/build.gradle

```diff
@@ -22,6 +28,28 @@ dependencies {
   testCompile group: 'commons-io', name: 'commons-io', version: '2.6'
   testCompile group: 'com.icegreen', name: 'greenmail', version: '1.5.8'
 
-  testCompile group: 'org.mockito', name: 'mockito-core', version: '2.21.0'
+  testCompile group: 'org.mockito', name: 'mockito-core', version: '2.15.0'
+
+  compile group: 'org.pitest', name: 'pitest', version: '1.3.2'
+  compile 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.3.0'
+  compile 'org.codehaus.sonar-plugins:sonar-pitest-plugin:0.5'
+
+  sonarqube {
+    properties {
+      property "sonar.host.url", "https://sonar.reform.hmcts.net/"
```

Comment:
this should not be here, its configured on jenkins and can be overridden on the command line with 
-Dsonar.host.url